Added Pinata page (#170)
Conflux-Chain/conflux-documentation: docs/general/build/nfts/pinata.md

---
id: pinata
title: Uploading files to IPFS using Pinata
sidebar_position: 1
keywords:
  - conflux
  - pinata
  - ipfs
---

# Uploading files to IPFS using Pinata

The initial phase in establishing an NFT smart contract involves preparing your NFT files. In this instance, the files will be uploaded to **[Pinata](https://www.pinata.cloud/)**, a service that keeps files from being purged on IPFS. If you haven't created an account yet, please do so.

## What is Pinata?

> Pinata is an NFT media management service that allows users to host, manage and share files of any kind on the blockchain of their choice. As an IPFS pinning service, we focus on giving both technical and non-technical creators a fast, easy, and reliable way to share content without limits.

Reviewer: should not be `... **we** focus on giving ...`

> For developers, Pinata is the easiest way to pin content to IPFS and build web3 applications without having to build and manage your own IPFS nodes. We do all the heavy lifting for you.

Reviewer: ```suggestion
For developers, Pinata is one of the easiest ways to pin content to IPFS and build web3 applications without having to build and manage your own IPFS nodes. Pinata do all the heavy lifting for you.
```

> For developers, Pinata iis one of the easiest ways to pin content to IPFS and build web3 applications without having to build and manage your own IPFS nodes. We do all the heavy lifting for you.

Reviewer: ```suggestion
For developers, Pinata is one of the easiest ways to pin content to IPFS and build web3 applications without having to build and manage your own IPFS nodes. It has done all the heavy lifting for developers.
```
